packages/ckeditor5-autoformat/src/autoformat.ts

```diff
@@ -201,6 +202,22 @@ export default class Autoformat extends Plugin {
 			blockAutoformatEditing( this.editor, this, /^---$/, 'horizontalLine' );
 		}
 	}
+
+	/**
+	 * TODO
```

Comment:
Remember to write API docs.

Reply:
I left that on purpose. If the API changes, this would be a waste of time.